Zend/zend_API.c

```diff
@@ -2933,6 +2934,80 @@ static zend_always_inline void zend_normalize_internal_type(zend_type *type) {
 	} ZEND_TYPE_FOREACH_END();
 }
 
+void zend_convert_internal_arg_info_type(zend_type *type)
+{
+	if (ZEND_TYPE_HAS_LITERAL_NAME(*type)) {
+		// gen_stubs.php does not support codegen for DNF types in arg infos.
+		// As a temporary workaround, we split the type name on `|` characters,
+		// converting it to an union type if necessary.
```

Comment:
I don't understand this comment. It's referring to DNF but is handling only union types.

*Edit*: I see this comes from a moved code block. Would still be nice to clarify. /cc @Girgias